README.md

```diff
@@ -44,15 +44,18 @@ Unfortunately, googletest and lapack are not as easy to install.  Many distros d
 #### (optional, one time only)
 1.  `mkdir -p [ROCBLAS_BUILD_DIR]/release/deps`
 2.  `cd [ROCBLAS_BUILD_DIR]/release/deps`
-3.  `cmake [ROCBLAS_SOURCE]/deps`
-4.  `make -j$(nproc) install`
+3.  `cmake -DBUILD_BOOST=OFF [ROCBLAS_SOURCE]/deps`
```

Comment:
I found that if you build Boost the build will fail. If you run the given command 
sudo apt install libboost-program-options-dev
and then build with -DBUILD_BOOST=OFF it will build correctly. This is what is done in the Jenkinsfile